server/BraveAlerterConfigurator.js

```diff
@@ -103,6 +104,28 @@ class BraveAlerterConfigurator {
 
         // If this is a OneSignal session (i.e. it isn't given the respondedByPhoneNumber) or if the SMS came from the session's respondedByPhoneNumber
         if (alertSession.respondedByPhoneNumber === undefined || alertSession.respondedByPhoneNumber === session.respondedByPhoneNumber) {
+          // Check for an invalid request to reset
+          if (
+            alertSession.alertState === CHATBOT_STATE.RESET &&
+            session.numberOfAlerts < helpers.getEnvVar('SESSION_MIN_NUMBER_OF_ALERTS_TO_ALLOW_RESET_REQUEST')
+          ) {
+            // Get language of client
+            const { language } = await db.getClientWithSessionId(alertSession.sessionId, pgClient)
+
+            const replacementReturnMessageToRespondedByPhoneNumber = t('resetRequestDeclined', { lng: language })
+            const replacementReturnMessageToOtherResponderPhoneNumbers = null // Don't send a message to the other responder phone numbers
```

Comment:
Looks good. This seems like it'll work. I like this `resetRequestDeclined` message for them. Very useful, I think.